analysis_options.yaml

```diff
@@ -8,7 +8,7 @@ analyzer:
     - .history/**
     - test/_data/**
     - test/mock_packages/**
-    - test/rules/**
+    #- test/rules/**
```

Comment:
I suspect that you intended to uncomment this.

Reply:
Apologies.

Reply:
No problem. Still LGTM.